src/components/OcAutocomplete.vue

```diff
@@ -683,9 +542,14 @@ export default {
         this.delayedResult = this.simpleItems
```

Comment:
Delayed result example seems broken

Reply:
I threw away the changes as since we want to deprecate it anyway it doesn't make too much sense to introduce changes to it now...

Reply:
Sure thing, then probably just remove the broken example? :)

Reply:
It works now, was broken due to new changes.